packages/browser/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -127,7 +127,7 @@ export default class OidcLoginHandler implements ILoginHandler {
     const OidcOptions: IOidcOptions = {
       issuer: options.oidcIssuer as URL,
       // TODO: differentiate if DPoP should be true
-      dpop: true,
+      dpop: options.tokenType === "DPoP",
```

Comment:
Are we really case-sensitive, or should this be a case-insensitive conditional...?

Reply:
Good point: TS users will have that checked for them (the expected type for tokenType is `"DPoP"|"Bearer"`, so you get auto-completion for this casing), but plain JS users won't necessarily notice that. Fixed in 03f44746ffd14f9f41c62c3e5b702ae386b4b1ea. In this case though, we rely on TS to prevent any other value to be passed. Should we throw on something with is neither dpop or bearer, casing aside ?

Reply:
Yeah, that sounds like a good idea.

Reply:
Yep - makes sense to throw alright.